examples/gno.land/r/gov/dao/v3/memberstore/memberstore.gno

```diff
@@ -0,0 +1,105 @@
+package memberstore
+
+import (
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/r/gov/dao/proxy"
+)
+
+var members MembersByTier
+var Tiers TiersByName
+
+const (
+	T1 = "T1"
+	T2 = "T2"
+	T3 = "T3"
+)
+
+func init() {
+	members = NewMembersByTier()
+
+	Tiers = TiersByName{avl.NewTree()}
+	Tiers.Set(T1, Tier{
+		InvitationPoints: 3,
+		Payment:          90000,
+		MinSize: func(membersByTier MembersByTier, tiersByName TiersByName) int {
+			return 70
+		},
+		MaxSize: func(membersByTier MembersByTier, tiersByName TiersByName) int {
+			return 0
+		},
+		BasePower: 3,
+		PowerHandler: func(membersByTier MembersByTier, tiersByName TiersByName) float64 {
+			return 3
+		},
+	})
+
+	Tiers.Set(T2, Tier{
+		InvitationPoints: 2,
+		Payment:          90000,
+		MaxSize: func(membersByTier MembersByTier, tiersByName TiersByName) int {
+			return membersByTier.GetTierSize(T1) * 4
```

Comment:
Isn't the [max size](https://gist.github.com/jaekwon/918ad325c4c8f7fb5d6e022e33cb7eb3#t2-membership-size) size(T1) * 2, or the ratio changed?